jobs/component_jobs.groovy

```diff
@@ -87,15 +100,19 @@ repos.each { Map repo ->
       }
 
       publishers {
-        def statuses = [['SUCCESS', 'pending'],['FAILURE', 'failure'],['ABORTED', 'error']]
-        postBuildScripts {
-          steps {
-            statuses.each { buildStatus, commitStatus ->
-              conditionalSteps {
-                condition {
-                  status(buildStatus, buildStatus)
-                  steps {
-                    shell curlStatus(buildStatus: buildStatus, commitStatus: commitStatus, jobName: name, repoName: repo.name)
+        if (isPR) {
+          def statuses = [['SUCCESS', 'pending'],['FAILURE', 'failure'],['ABORTED', 'error']]
+          postBuildScripts {
+            onlyIfBuildSucceeds(false)
+            steps {
+              statuses.each { buildStatus, commitStatus ->
+                conditionalSteps {
+                  condition {
+                    status(buildStatus, buildStatus)
+                    steps {
+                      shell StatusUpdater.curlStatus(
```

Comment:
I don't think this will do what you expect it to do because everything in the `steps { shell ...` block will be sent to be held within the Jenkins job config and executed when the job is run. In other words, `StatusUpdater.curlStatus(...` will be called in bash on the Jenkins slave when the job executes.